templates/facets/facets-item-list--checkbox--localgov-directories-facets.html.twig

```diff
@@ -0,0 +1,59 @@
+{#
+/**
+ * @file
+ * Default theme implementation for a facets item list.
+ *
+ * Available variables:
+ * - items: A list of items. Each item contains:
+ *   - attributes: HTML attributes to be applied to each list item.
+ *   - value: The content of the list element.
+ * - title: The title of the list.
+ * - list_type: The tag for list element ("ul" or "ol").
+ * - wrapper_attributes: HTML attributes to be applied to the list wrapper.
+ * - attributes: HTML attributes to be applied to the list.
+ * - empty: A message to display when there are no items. Allowed value is a
+ *   string or render array.
+ * - context: A list of contextual data associated with the list. May contain:
+ *   - list_style: The ID of the widget plugin this facet uses.
+ * - facet: The facet for this result item.
+ *   - id: the machine name for the facet.
+ *   - label: The facet label.
+ *
+ * @see facets_preprocess_facets_item_list()
+ *
+ * @ingroup themeable
+ */
+#}
+<div class="facets-widget- {{- facet.widget.type -}}">
+  {% if facet.widget.type %}
+    {%- set attributes = attributes.addClass('item-list__' ~ facet.widget.type) %}
+  {% endif %}
+  {% if items or empty %}
+  {%- if title is not empty -%}
+    <h3>{{ title }}</h3>
+  {%- endif -%}
+
+  {%- if items -%}
+  {#
+    The js-facets-checkbox-links class is only meant for those ul/ol elements
+    whose *direct* children are facet items and not titles or other ul/ol.
+
+    @see facets/templates/facets-item-list.html.twig
+    @see Drupal.facets.makeCheckboxes()
+  #}
+  {% set top_list_type_attr = create_attribute(attributes.toArray()).removeClass('js-facets-checkbox-links') %}
+  <{{ list_type }}{{ top_list_type_attr }}>
+  {%- for group in items -%}
+    <li{{ group.attributes }}><h4>{{ group.title }}</h4></li>
```

Comment:
+1.  Perhaps the [default template](https://github.com/localgovdrupal/localgov_directories/blob/ef3b48817f4d8121fee7944512c34cf76f4c350b/templates/facets-item-list--links--localgov-directories-facets.html.twig#L47) should do this too.  What do you think?

Reply:
I have removed the theme template file as the changes have been moved to the default module file